Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get -y install python python-pip libpq-dev python-dev && \
+    apt-get clean
+
+COPY init_db.py requirements.txt runserver.py /opt/dashboardsly/
+COPY dashboardsly /opt/dashboardsly/dashboardsly/
+
+RUN cd /opt/dashboardsly && pip install -r requirements.txt
+
+EXPOSE 80
+CMD cd /opt/dashboardsly; gunicorn runserver:app --log-file=- -b 0.0.0.0:80
```

Comment:
Yeah.  And I've had a stern talk to my editor about doing this automatically in future.